src/st/lib/unk_2FA80.c

```diff
@@ -910,17 +788,12 @@ INCLUDE_ASM("st/lib/nonmatchings/unk_2FA80", func_us_801B1200);
 
 INCLUDE_ASM("st/lib/nonmatchings/unk_2FA80", func_us_801B12D0);
 
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACDE0);
-
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACDEC);
-
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACDF4);
-
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACE00);
-
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACE0C);
-
-INCLUDE_RODATA("st/lib/nonmatchings/unk_2FA80", D_us_801ACE18);
+const char D_us_801ACDE0[] = "\003Sell gem";
+const char D_us_801ACDEC[] = "\006Exit";
+const char D_us_801ACDF4[] = "\001Sound test";
+const char D_us_801ACE00[] = "\001Enemy list";
+const char D_us_801ACE0C[] = "\003Tactics";
+const char D_us_801ACE18[] = "\003Buy item";
```

Comment:
Is it possible these are `struct { u8, char[0] }` using the trailing array idiom (e.g. `GfxBank`, `SpriteParts`, etc.)?